js/review-data-types-1.js

```diff
@@ -0,0 +1,81 @@
+// integer variables
+
+var a= 100;
+console.log(a);
+var b=200;
+console.log(b);
+var c=50;
+console.log(c);
+var d=150;
+console.log(d);
+var e=300;
+console.log(e);
+
+// string variables
+
+var a= '100';
+console.log(a);
+var b='200';
+console.log(b);
+var c='50';
+console.log(c);
+var d='150';
+console.log(d);
+var e='300';
+console.log(e);
+
+// boolean variables
+
+var a = true;
+var b = false;
+
+console.log(a);
+console.log(b);
+
+// array variables
+
+var arr1=['pravallika', 'prasad', 'sahasra'];
+var arr2=['hello', 'world'];
+var arr3=['Images','videos','maps'];
+var arr4=['purple','green',10,20];
+var arr5=[10 ,20, 30, 40, 50];
+
+console.log(arr1);
+console.log(arr2);
+console.log(arr3);
+console.log(arr4);
+console.log(arr5);
+
+//object variables
+
+var phone={
+    make:'Apple',
+    model:'6s',
+    year:2017
+};
+console.log(phone);
+var person={
+    firstName:'Pravalika',
+    lastname:'Ammineni',
```

Comment:
Follow same naming convention for all properties of an object. Since firstName was camelCase, use the same here. So lastName